docs/guided_fuzzing.md

```diff
@@ -0,0 +1,25 @@
+# Challenges of guided fuzzing
```

Comment:
remove this file + content please. it is out of context, outdated, and I think does not help anyone.